retroshare-service/CMakeLists.txt

```diff
@@ -113,6 +113,23 @@ target_link_libraries(${PROJECT_NAME} PRIVATE retroshare)
 
 ################################################################################
 
+# Add RNP build directories to linker search path
+target_link_directories(${PROJECT_NAME} PRIVATE
+    ${PROJECT_SOURCE_DIR}/../supportlibs/librnp/Build/src/lib
+    ${PROJECT_SOURCE_DIR}/../supportlibs/librnp/Build/src/libsexpp
+)
+
+# Link libraries by name, including RNP libs and dependencies first
+target_link_libraries(${PROJECT_NAME} PRIVATE
+    rnp             # Link by name
+    sexpp
+    botan-3
```

Comment:
-- Looking for BOTAN_HAS_DL_PUBLIC_KEY_FAMILY - not found
CMake Error at src/lib/CMakeLists.txt:178 (message):
  A required botan feature is missing: DL_PUBLIC_KEY_FAMILY

if compiled with qmake normal gui version

Reply:
botan is installed via brew botan (version 3.)

Reply:
i was able to compile it, but only with botan-2